core/src/main/java/io/kestra/core/docs/JsonSchemaGenerator.java

```diff
@@ -525,6 +526,54 @@ public CustomDefinition provideCustomSchemaDefinition(ResolvedType javaType, Sch
                     }
                 }
             });
+        } else {
+            builder.forTypesInGeneral().withTypeAttributeOverride((collectedTypeAttributes, scope, context) -> {
```

Comment:
maybe extract this in a named function for clarity if possible ?
or add a comment to explain? 
without this commit/issue it seems hard to understand what the code does